src/agent/onefuzz-agent/src/tasks/report/crash_report.rs

```diff
@@ -36,6 +36,9 @@ pub struct CrashReport {
     pub task_id: Uuid,
 
     pub job_id: Uuid,
+
+    pub scariness_index: Option<u32>,
```

Comment:
I suggest we call these `scariness_score` and `scariness_description`, to match the compiler-rt language.